controller/planner_backlog_blackbox_test.go

```diff
@@ -221,12 +220,11 @@ func (rest *TestPlannerBacklogBlackboxREST) TestSuccessEmptyListPlannerBacklogWo
 			rest.T().Error(err)
 		}
 		spaceID = p.ID
-		userActive := false
 		parentIteration = &iteration.Iteration{
 			Name:       "Parent Iteration",
 			SpaceID:    spaceID,
 			State:      iteration.IterationStateNew,
-			UserActive: &userActive,
+			UserActive: false,
```

Comment:
Remove this line, since `UserActive` will automatically be `false`.

Reply:
Removed it here b92020c